### ISSUE TITLE
Certificate is invalid

### ISSUE THREAD

**Issue by Samsonkin** [Rambalac/GMaster#29]
I'm trying to install the version from https://github.com/Rambalac/GMaster/releases

<img src="https://i.ibb.co/FBsVJJy/2019-01-06-14-14-30-Certificate.png" alt="2019-01-06-14-14-30-Certificate" border="0">

**Comment by rogerclarkmelbourne** [comment]
I had the same issue.

I tried following some tutorials about installing the certificate into W10, but nothing seemed to work, perhaps there is now way to install this now, as W10 won't ever except an expired cert.

BTW. I also tried to recompile from source in VS2019 but it won't build the GMaster solution, because Visual Studio says a project properties file is missing.

This could be something specific to VS2019. I tried building using VS2017 under W7 but could not open the GMaster project as it seems to be a W10 only thing.

It would be better if this was not a W10 app, but was a simple exe, as there would not be all the problems with certificates etc. 

But I appreciate that since there has been no updates on this for over a year, that this project is effectively dead, as there is no one to maintain it.

**Comment by KonTy** [comment]
Yeah unfortunately this project has been abandoned, so do not expect any help here. Eventough the author spent a lot of time with Wireshark reverse engineering Panasonic protocol. Too bad so much work was done for nothing. The argument was that MS doesn't make phones anymore but the thing is that you still can use this on Surface and other portable tablet devices.

**Comment by KonTy** [comment]
> It would be better if this was not a W10 app, but was a simple exe, as there would not be all the problems with certificates etc.

In any case with the source code available you can fix all these problems yourself. including the cert problem you can regenerate/replace that cert with your own. Same applies to the properties file there isn't much stored in those files if could be reverse engineered :)

**Comment by IdrisTheDragon** [comment]
You can still find the App on the Windows 10 App store under "LMaster" and last I checked a few months ago, it still works fine: https://www.microsoft.com/en-gb/p/lmaster/9nc2w8kc526f?activetab=pivot:overviewtab

Also a similar project that uses Java and a simple html webpage that I know of is this one, which might be easier to update and maintain yourself:
https://github.com/peci1/lumix-link-desktop
http://www.personal-view.com/talks/discussion/6703/control-your-gh3-from-a-web-browser-now-with-video-/p1

**Comment by pedzed** [comment]
I'm planning to make a cross-platform (Windows/Linux/macOS) application myself in the future. I'm therefore closely observing repositories.

If you would like to help me test your Lumix camera model(s) in the future, please mail me at `dev (at-sign) pedzed.com`

**Comment by pedzed** [comment]
@KonTy where can I read more about the Wireshark results?

**Comment by IdrisTheDragon** [comment]
@pedzed if you have a Lumix camera you can capture packets between camera and the app yourself and view them in wireshark quite easily, I had play about with this a while ago.

I used this app on my phone to capture the packet data from the genuine Lumix app, it's slightly easier than using wireshark directly as you can easily filter only the app specific packets, you can also export the data and view it in wireshark: https://play.google.com/store/apps/details?id=app.greyshirts.sslcapture

**Comment by rogerclarkmelbourne** [comment]
I presumed someone had simply decompiled the Android app.

My Android phone runs a custom ROM so I can easily get to the APK

The command set isn't a big deal, its fairly easy.
The streaming video for the live view will be more difficult.

**Comment by rogerclarkmelbourne** [comment]
BTW. I did try to recompile but I don't know enough about W10 "apps".

I do however program in C# using an older copy of Visual Studio for my day job.
But I only build exes
